src/coreclr/nativeaot/Runtime/i386/WriteBarriers.asm

```diff
@@ -48,7 +48,7 @@ UPDATE_GC_SHADOW macro BASENAME, DESTREG, REFREG
     jb      &BASENAME&_UpdateShadowHeap_PopThenDone_&DESTREG&_&REFREG&
     add     DESTREG, [g_GCShadow]
     cmp     DESTREG, [g_GCShadowEnd]
-    ja      &BASENAME&_UpdateShadowHeap_PopThenDone_&DESTREG&_&REFREG&
+    jae     &BASENAME&_UpdateShadowHeap_PopThenDone_&DESTREG&_&REFREG&
```

Comment:
This is for i386. Should we also update the amd64 too?

https://github.com/dotnet/runtime/blob/a877553633aa9286bfcc8012870ad0127a3ab997/src/coreclr/nativeaot/Runtime/amd64/WriteBarriers.asm#L46

Reply:
I have already updated that line in this PR.

Reply:
Ugh. Sorry. Misread arm64 vs amd64.